src/modules/rppt_tensor_geometric_augmentations.cpp

```diff
@@ -153,6 +153,33 @@ RppStatus rppt_crop_mirror_normalize_host(RppPtr_t srcPtr,
                                                 roiType,
                                                 layoutParams);
     }
+    else if ((srcDescPtr->dataType == RpptDataType::U8) && (dstDescPtr->dataType == RpptDataType::F32))
+    {
+        crop_mirror_normalize_u8_f32_host_tensor(static_cast<Rpp8u*>(srcPtr) + srcDescPtr->offsetInBytes,
+                                                 srcDescPtr,
+                                                 (Rpp32f*) (static_cast<Rpp8u*>(dstPtr) + dstDescPtr->offsetInBytes),
+                                                 dstDescPtr,
+                                                 meanTensor,
```

Comment:
are the mean and std values different for each image in the tensor?

Reply:
Hi Rajy,
Yes, the mean and std values can be different for each image in the tensor